colossalai/shardformer/flashattention/flash_attention_opt.py

```diff
@@ -0,0 +1,107 @@
+from typing import List, Optional, Tuple, Union
+
+import torch
+import torch.nn as nn
+from torch.distributed import ProcessGroup
+from xformers.ops import memory_efficient_attention as me_attention
```

Comment:
Write a try-except block to tell the user that `xformers` should be installed in order to use flash atttention.

Reply:
You can also import this only inside the class itself.